include/pmacc/identifier/value_identifier.hpp

```diff
@@ -50,3 +50,8 @@
     identifier(                                                                                                       \
         name, using type = in_type; static HDINLINE type getValue()                                                   \
         { return in_default; } static std::string getName() { return std::string(#name); })
+
+#define value_identifier_constexpr(in_type, name, in_default)                                                         \
```

Comment:
PLease add documentation.
Use in the comment for `value_identifyer` a group comment`@{`

and than for `value_identifier_constexpr`

```
/** getValue() is defined constexpr
  * @}
  */
```

Reply:
This still looks unresolved to me. Is this correct @pordyna @psychocoderHPC

Reply:
is is solved now